src/lib/migration.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Sets audit configuration values for a database migration.
+ * @param {Object} queryInterface - The Sequelize Query Interface object.
+ * @param {Object} transaction - The Sequelize Transaction object.
+ * @param {string} sessionSig - A unique identifier for the current session.
+ * @param {string} [auditDescriptor='RUN MIGRATIONS'] - A description of the audit event.
+ * @param {string} [loggedUser='0'] - The user ID associated with the audit event.
+ * @returns {Promise} - A Promise that resolves when the audit configuration is set.
+ */
+const prepMigration = async (
+  queryInterface,
+  transaction,
+  sessionSig,
+  auditDescriptor = 'RUN MIGRATIONS',
+  loggedUser = '0',
```

Comment:
These functions are really smart ideas. My only question is whether we'd ever want to change "loggedUser".

Reply:
audit log defaults to -1 when its not set at all. we have been using 0 for migrations